src/workarounds/forwarddiff_rules.jl

```diff
@@ -0,0 +1,161 @@
+import ForwardDiff
+import AbstractFFTs
+
```

Comment:
Quite a bit of these workarounds are not covered by tests at the moment, which bothers me a bit. I know this will not last forever (hopefully), but if you can think of an easy way to increase coverage here @niklasschmitz (like another nice example or so) that would be good.